src/main/java/model/EventModel.java

```diff
@@ -72,25 +80,54 @@ public boolean updateEvent(EventBean event) {
 	}
 	
 	public boolean deleteEvent(String id) {
-		return daoFactory.getEventDao().deleteEvent(Integer.parseInt(id));
+		if(checkInteger(id)) {
+			return daoFactory.getEventDao().deleteEvent(Integer.parseInt(id));
+		}
+		else {
+			return false;
+		}
 	}
 	
 	public ArrayList<ParticipantBean> getParticipantList(String id) {
 		ArrayList<ParticipantBean> result = new ArrayList<ParticipantBean>();
-		ResultSet data = daoFactory.getEventDao().getParticipants(Integer.parseInt(id));
-		if(data != null) {
-			try {
-				while(data.next()) {
-					ParticipantBean participant = new ParticipantBean();
-					participant.setEventId(data.getInt("id_event"));
-					participant.setUserId(data.getInt("id_user"));
-					participant.setUsername(data.getString("username"));
-					participant.setStatusId(data.getInt("id_status"));
-					participant.setStatus(data.getString("status"));
-					result.add(participant);
+		if(checkInteger(id)) {
+			ResultSet data = daoFactory.getEventDao().getParticipants(Integer.parseInt(id));
+			if(data != null) {
+				try {
+					while(data.next()) {
+						ParticipantBean participant = new ParticipantBean();
+						participant.setEventId(data.getInt("id_event"));
+						participant.setUserId(data.getInt("id_user"));
+						participant.setUsername(data.getString("username"));
+						participant.setStatusId(data.getInt("id_status"));
+						participant.setStatus(data.getString("status"));
+						result.add(participant);
+					}
+				} catch (SQLException e) {
+					e.printStackTrace();
+				}
+			}
+		}
+		return result;
+	}
+	
+	public boolean resolveParticipant(Map<String, String[]> parameters) {
+		boolean result = false;
+		if(parameters != null && !parameters.isEmpty() && parameters.containsKey("event") && parameters.containsKey("user") && parameters.containsKey("action")) {
+			if(checkInteger(parameters.get("event")[0]) && checkInteger(parameters.get("user")[0]) && checkInteger(parameters.get("action")[0])) {
```

Comment:
It would be preferable to use body instead of header parameters as you can avoid most of these checks with the help of gson and it is more manageble.